tests/pyfunc/docker/test_docker.py

```diff
@@ -0,0 +1,129 @@
+import difflib
+import os
+import shutil
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Optional
+from unittest import mock
+
+import pytest
+import sklearn
+import sklearn.neighbors
+
+import mlflow
+from mlflow.models import Model
+from mlflow.models.docker_utils import build_image_from_context
+from mlflow.models.flavor_backend_registry import get_flavor_backend
+from mlflow.utils import PYTHON_VERSION
+from mlflow.utils.env_manager import VIRTUALENV
+from mlflow.version import VERSION
+
+from tests.pyfunc.docker.conftest import RESOURCE_DIR, get_released_mlflow_version
+
+
+def assert_dockerfiles_equal(actual_dockerfile_path: Path, expected_dockerfile_path: Path):
+    actual_dockerfile = actual_dockerfile_path.read_text().replace(
+        VERSION, get_released_mlflow_version()
+    )
+    expected_dockerfile = (
+        expected_dockerfile_path.read_text()
+        .replace("$MLFLOW_VERSION", get_released_mlflow_version())
```

Comment:
```suggestion
        .replace("${{ MLFLOW_VERSION }}", get_released_mlflow_version())
```

Can we use `${{ ... }}` as a placeholder? `$...` looks like it's an environment variable.

Reply:
Makes sense! Will replace them.